docs/tutorials/image_classification/hpo.md

```diff
@@ -124,6 +124,10 @@ Here is an example of using an early stopping scheduler :class:`autogluon.schedu
 
 ```{.python .input}
 search_strategy = 'hyperband'
+scheduler_options = {
+    'grace_period': 1,
```

Comment:
add brief comment in the tutorial explaining what is 'grace_period', 'reduction_factor', 'brackets'.

Reply:
OK